pkg/commands/artifact/run.go

```diff
@@ -492,20 +492,31 @@ func initScannerConfig(opts flag.Options, cacheClient cache.Cache) (ScannerConfi
 		log.Logger.Debugf("Vulnerability type:  %s", scanOptions.VulnType)
 	}
 
+	var downloadedPolicyPaths []string
+	var disableEmbedded bool
+	downloadedPolicyPaths, err := operation.InitBuiltinPolicies(context.Background(), opts.CacheDir, opts.Quiet, opts.SkipPolicyUpdate)
+	if err != nil {
+		log.Logger.Errorf("Falling back to embedded policies: %s", err)
```

Comment:
It shows this error even when `--skip-policy-update` is specified on the first run.

```
$ trivy config --skip-policy-update .
2023-01-11T14:33:20.119+0200    INFO    No downloadable policies were loaded as --skip-policy-update is enabled
2023-01-11T14:33:20.119+0200    ERROR   Falling back to embedded policies: No downloadable policies were loaded as --skip-policy-update is enabled
```

I think only unexpected things should be shown as an error. If policies cannot be pulled due to network or storage issues, it should be an error. What do you think? 
If you don't think embedded policies should be used even with `--skip-policy-update`, which means they should have downloaded policies locally, the error is ok because using embedded policies is unexpected.

Reply:
Right, I think the former is better (only error in unexpected situations). I think that's easy to check with err != nil and false skip update, that should signal that user intended to download new policies but failed to do so. 

I added that check in: https://github.com/aquasecurity/trivy/pull/3015/commits/ac1eef6ef4f49022e736583da54c715f55b9d2f2